[CS2103T-T11-4] TravelPal

teoha opened the pull request:
@ang-zeyu @arjunwangty @eizon05 @SakshiPradyumn

dalsontws commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
 
-|`* * *` |user |add a new person |
+[appendix]
+== Use Cases
```
For use cases, the System, Actor and Preconditions are not clearly defined. Good to have that defined clearly before each  use case to avoid ambiguity

dalsontws commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
 
-*Product Name*
+[appendix]
+== Glossary
```
Would be good to have 'Trip Manager' in your Glossary, slightly unclear as to what it is

keiteo commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
-== Product Survey
+6.	Application does not depend on online resources to operate
+
+7.	Products is not required to make decisions for the user
```
Might want to make this point clearer? Not too sure what do the products refer to. Also, there's a minor grammar error to be changed!

keiteo replied:
Might want to make this point clearer? Not too sure what do the products refer to. Also, there's a minor grammar error to be changed!

dalsontws commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
+
+Steps 5a1-5a2 are repeated until no clashes occur between trips
+
+<span class="underline">Use case: UC2 – Edit Trip</span>
```
Formatting error!

dalsontws commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
-=== Use case: Delete person
+6.  TravelPal shows the list of **Trips**
+    
+    Use case ends.
```
The "Use case ends." formatting is different for each use case, should be consistent

LeowWB commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
+image::us1.PNG[]
+image::us2.PNG[]
+image::us3.PNG[]
+image::us4.PNG[]
```
User stories generally in correct format, and benefits do match the functions. You might want to re-look the last few user stories, particularly the ones in `us4.PNG'.

* Sight-seeing enthusiast - maybe make it universal and not just in Singapore?
* Travel/tour guide, teacher - can combine into single user story (since both are planning trips for others)?
* Salesperson - feels like the user story not relevant to the user.
* Accountant - I think "decide if the expenses are appropriate" isn't very well-defined.

You might want to add the benefits for more user stories too, I think it would help to flesh them out!

Otherwise, the other images are generally okay! Keep up the good work 👍👍👍

keiteo commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
 
-[[private-contact-detail]] Private contact detail::
-A contact detail that is not meant to be shared with others
+5. 	A novice user should be able to navigate without prior experience
```
Are these really NFR? Seems like part of a user story!

keiteo commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
-.  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+1.  Should work on any [mainstream OS] as
```
Formatting for "mainstream OS"

keiteo commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
 
-* ...
-* ...
+**Mainstream OS** - Windows, Linux, Unix, OS-X
```
Do take note that this glossary is part of the developer's guide. Are these terms really necessary?

keiteo commented on docs/DeveloperGuide.adoc:
```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
+    long as it has Java 11 or above installed.
 
-_{More to be added}_
+2.  A user with above average typing speed for regular English text
+    (i.e. not code, not system admin commands) should be able to
+    accomplish most of the tasks faster using commands than using the
+    mouse.
```
May want to make this a bit more concrete so that it's easier for developers to test!

teoha committed:
Fixed checkstype error from resolving merge conflicts

ang-zeyu committed:
Fixes:
- Wrong alignment for DiaryLine photoCardsDisplay when graphic is to be placed on the left
- Refactored magic number for padding in DiaryLine to class level constant

ang-zeyu committed:
Merge branch 'branch-diary' into branch-diary-user-guide

ang-zeyu committed:
Changes (diary UserGuide):
- add complete introductory sections to diary section
- add command sections (without intended images currently) to diary section

ang-zeyu committed:
Merge branch 'branch-diary-user-guide' into branch-diary

arjunwangty committed:
Update diagrams

ang-zeyu committed:
Add images for many more diary user guide sections

ang-zeyu committed:
Merge branch 'branch-diary-user-guide' into branch-diary

ang-zeyu committed:
Add newline at userguide.adoc eof

ang-zeyu committed:
Merge branch 'branch-diary-user-guide' into branch-diary

ang-zeyu committed:
Small fix for DiaryPage SwapRightToGallery not swapping the photoList when the page is flipped

teoha committed:
Changed page to adhere to SRP and refactored it so pages requiring change to ui implements UiPageConsumer

arjunwangty committed:
add currency feature

arjunwangty committed:
Merge branch 'master' into Branch-Expense-Manager

aang114 committed:
Merge pull request #82 from arjunwangty/Branch-Expense-Manager

add currency feature

arjunwangty committed:
Merge branch 'master' into branch-itinerary-improve-ui

arjunwangty committed:
Update EventsPage.java

arjunwangty committed:
Update EventsPage.java

arjunwangty committed:
Merge pull request #81 from teoha/branch-itinerary-improve-ui

Branch itinerary improve ui

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

ang-zeyu committed:
Changes (docs)
- add tags to user and developer guide diary sections for display in project portfolio
- add initial draft for project portfolio page
- change link in AboutUs to link to project portfolio page

ang-zeyu committed:
Merge branch 'branch-project-portfolio' into branch-diary

ang-zeyu committed:
Merge branch 'master' into branch-diary
- fix ImageChooser import path conflict since moving from parser.diary.fileutil to commons.util
- readd overloaded constructor for CommandResult to maintain original CommandResult constructor use
- fix minor merge conflict in PageStatus

# Conflicts:
#	src/main/java/seedu/address/model/appstatus/PageStatus.java

ang-zeyu committed:
Merge pull request #84 from ang-zeyu/branch-diary

Documentation and more tests

teoha committed:
Changed:
Ui.png

ang-zeyu committed:
Merge pull request #86 from teoha/branch-ui-png-update

Update Ui Image

ang-zeyu committed:
Fix MainApp.java version number

ang-zeyu committed:
Changes (User Guide):
- finish adding step-by-step image instructions for diary sections
- remove irrelavant / deprecated details in various user guide sections

ang-zeyu committed:
Merge branch 'branch-diary-user-guide' into branch-diary

ang-zeyu committed:
Fixes:
- fix newline not being appended during setDiaryText call from gui due to ArgumentMultimap trimming newlines
- incorrect error message thrown for CreateDiaryEntryParser
- incorrect prefix used in preferences page for window x and y position when firing command from PreferencesPage
- removed unnecessary fillPage call in PreferencesPage initFormWithModel()

teoha committed:
Changed:
1. Fixed bug where dates could be added outside containing occurrence
2. Fixed bug where start dates could be after end dates
3. Reverted sorting of internal list
4. Fixed ui bug in days page where the scrollbar would not render

teoha committed:
Fixed checkstyle errors

teoha committed:
Merge branch 'master' into branch-fix-itinerary-bugs

ang-zeyu committed:
Merge pull request #89 from teoha/branch-fix-itinerary-bugs

Branch fix itinerary bugs

ang-zeyu committed:
Changes:
- Improved diary error messages ( distinguish MESSAGE_MISSING_FIELDS and MESSAGE_INVALID_COMMAND_FORMAT )

Fixes:
- fixed AppendDiaryEntryParser not stripping leading whitespace

Changes (Test):
- Add more tests (some test factories) for diary parsers (> 90% coverage)
- Fix DiaryTestUtil generateRandomText only generating numbers

ang-zeyu committed:
Merge branch 'master' into branch-diary

teoha committed:
Added:
1. stashed tests with checkstyle errors

ang-zeyu committed:
Small individual project portfolio page update

ang-zeyu committed:
Merge branch 'branch-project-portfolio' into branch-diary

ang-zeyu committed:
Add icon attributions

ang-zeyu committed:
Merge pull request #90 from ang-zeyu/branch-diary

More diary tests and documentation

ang-zeyu committed:
fix globe_icon.png link

teoha committed:
Changed:
1. Reversed disabling create command in events page

arjunwangty committed:
Update expense manager/ trip manager UG

teoha committed:
UG

teoha committed:
Merge branch 'master' into branch-itinerary-guides

teoha committed:
Merge pull request #91 from arjunwangty/Branch-User-Guide

Update expense manager/ trip manager UG

teoha committed:
Merge branch 'master' into branch-itinerary-guides

teoha committed:
Merge pull request #92 from teoha/branch-itinerary-guides

Branch itinerary guides

ang-zeyu committed:
Merge branch 'master' into branch-diary

ang-zeyu committed:
Changes (documentation):
- Revert addition of globe-icon
- Remove site-seedu flag in build.gradle

teoha committed:
Changed:
1. Days page formatting

teoha committed:
Changed:
1. Added the edit event command back
2. Removed the edit day command

teoha committed:
Merge branch 'master' into branch-fix-itinerary-bugs

teoha committed:
Revert "Changed:"

This reverts commit a8503a2d21fc74dd9c95d167594cf184f2a3872b.

teoha committed:
Edited parser

teoha committed:
Fixed bug where daypage bar could not be seen

teoha committed:
Merge branch 'branch-fix-itinerary-bugs' of https://github.com/teoha/main into branch-fix-itinerary-bugs

teoha committed:
Added:
1. UG for more itinerary features

teoha committed:
Changed:
Itinerary UI beautified

teoha committed:
Changed:
1. Removed functionality to add or delete days

teoha committed:
Changed:
1. Solved editing trip dates bugs

arjunwangty committed:
Fix bugs, Add preset symbol options

teoha committed:
Updated user guides and merge

teoha committed:
Changed:
1. Changed names of labels to be clearer

teoha committed:
Added:
Class diagram for editTripPage

ang-zeyu committed:
Changes:
- Removed preferences page implementations - fixes #110, #111, #113, #133, #134, #135, #144,
- Enforce global minimum window width and height of 640 by 480, added relavant section in user guide

Fixes:
- fixes #108 - editing trip wipes diary
- fixes #105 - Out of bounds day not rejected
- fixes #123 - Help button in trip manager not working
- fixes #143, #109, #117, #132 - leftover step from ab3 in user guide quick start
- fixed #116 - help command not working
- fixes #147 - grammar errors in error messages
- fixes #105 - able to add diary entry beyond the duration of the trip
- fixed AboutUs.adoc link

ang-zeyu committed:
Merge pull request #94 from teoha/branch-fix-itinerary-bugs

Branch fix itinerary bugs

teoha committed:
Merge branch 'master' into branch-beautify-itinerary

ang-zeyu committed:
Merge branch 'master' into Branch-Expense-Manager

ang-zeyu committed:
Merge pull request #149 from teoha/branch-beautify-itinerary

Branch beautify itinerary and fix bugs

ang-zeyu committed:
Merge branch 'master' into Branch-Expense-Manager

ang-zeyu committed:
Merge branch 'master' into branch-diary

# Conflicts:
#	src/main/java/seedu/address/logic/commands/trips/edit/EditTripFieldCommand.java

ang-zeyu committed:
Changes:
- Slightly changed diary day out of bounds message

arjunwangty committed:
Fix more bugs, merge expense manager / expense manager days page

arjunwangty committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-4/main into Branch-Expense-Manager

arjunwangty committed:
Merge branch 'Branch-Expense-Manager' of https://github.com/arjunwangty/main into Branch-Expense-Manager

ang-zeyu committed:
Merge pull request #150 from arjunwangty/Branch-Expense-Manager

Fix bugs, Add preset symbol options

ang-zeyu committed:
Merge branch 'master' into branch-diary

# Conflicts:
#	src/main/java/seedu/address/ui/expenditure/DailyExpenditurePage.java
#	src/main/java/seedu/address/ui/expenditure/ExpensesListPage.java

ang-zeyu committed:
Stage missing file removals during merge from master not staged during the merge commit

ang-zeyu committed:
Remove handlePreferences in ExpensesPage

teoha committed:
Added:
1. added back mistakenly edited modelClassDiagram class.
2. updated DG

arjunwangty committed:
Fix more bugs, sync Event and Expenditure

arjunwangty committed:
Merge branch 'master' into Branch-Expense-Manager

ang-zeyu committed:
Merge pull request #152 from arjunwangty/Branch-Expense-Manager

Fix more bugs, sync Event and Expenditure

ang-zeyu committed:
Merge branch 'master' into branch-diary

ang-zeyu committed:
Merge pull request #151 from ang-zeyu/branch-diary

Fix dry PE common and diary issues

ang-zeyu committed:
Changes:
- Removed unused fxmls
- Some code cleanup

Fixes:
- #146, #97 partial fix

ang-zeyu committed:
Merge branch 'master' into branch-diary

ang-zeyu committed:
Small style fix

arjunwangty committed:
Update user guide, disable duplicated event names, fix bugs

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

ang-zeyu committed:
Changes:
- Improved EditTripPage ui
- Cleanup EditTripPage code
- Some small diary ui tweaks

ang-zeyu committed:
Merge pull request #154 from arjunwangty/Branch-Expense-Manager

Update user guide, disable duplicated event names, fix bugs

ang-zeyu committed:
Merge branch 'master' into branch-diary

# Conflicts:
#	src/main/resources/view/trips/EditTripPage.fxml

ang-zeyu committed:
Changes:
- Style standardisations across trip manager and edit trip page

Fixes:
- Diary line numbers showing 0 when there isn't a line

teoha committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-4/main into branch-pe-bugfix

# Conflicts:
#	src/main/java/seedu/address/ui/itinerary/ItineraryPage.java

teoha committed:
Changed:
1. navbars now position correctly
2. Budget overflow bug for edit event/day page fixed
3. Removed placeholders for bookings and inventory for demo purposes

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

ang-zeyu committed:
Merge pull request #155 from teoha/branch-pe-bugfix

Branch pe bugfix

ang-zeyu committed:
Merge branch 'master' into branch-diary

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

teoha committed:
Changed:
1. Fixed bug not creating correct number of days

teoha committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-4/main into branch-itinerary-tests

teoha committed:
Changes:
1. Added description to events
2. description label added to the events page

teoha committed:
Merge branch 'branch-pe-bugfix' into branch-itinerary-tests

teoha committed:
Added:
1. Tests for models complete less events until all other features are finalized

ang-zeyu committed:
Changes:
- EditTripFieldCommand displays MESSAGE_USAGE inside MESSAGE_NOT_EDITED too
- Minor edit event page ui change

teoha committed:
Changed:
1. CommandUtil stubs to make testing easier
2. Tests still broken

teoha committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-4/main into branch-itinerary-guides

# Conflicts:
#	docs/UserGuide.adoc
#	src/main/resources/view/itinerary/days/DaysPage.fxml

teoha committed:
Changed:
1. Commented problematic budget test

teoha committed:
Changed:
1. Checkstyle errors

teoha committed:
eof

arjunwangty committed:
Add two type of expenditures

SakshiPradyumn committed:
Version 1 Booking

teoha committed:
Changed:
1. Bugfixes for pe

teoha committed:
Merge pull request #153 from ang-zeyu/branch-diary

Fix budget value overflow

arjunwangty committed:
Merge branch 'master' into Branch-Expense-Manager

teoha committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-4/main into branch-itinerary-tests

arjunwangty committed:
no message

arjunwangty committed:
Merge branch 'master' into Branch-Expense-Manager

arjunwangty committed:
Merge branch 'Branch-Expense-Manager' of https://github.com/arjunwangty/main into Branch-Expense-Manager

ang-zeyu committed:
Merge branch 'master' into branch-itinerary-guides

teoha committed:
Added:
1. Image for days

teoha committed:
Merge branch 'branch-pe-bugfix' into branch-itinerary-tests

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

arjunwangty committed:
Update docs and namings

teoha committed:
Added:
1. All tests for itinerary
Changes:
1. Fixes to many bugs

arjunwangty committed:
Updates UI, add sorting feature

aang114 committed:
Merge pull request #160 from arjunwangty/Branch-Expense-Manager

Add two type of expenditures

arjunwangty committed:
Merge branch 'master' into branch-itinerary-guides

Commit:
Added Done Feature & Linked Inventory Items to Events

arjunwangty committed:
Merge pull request #158 from teoha/branch-itinerary-guides

Branch itinerary guides

teoha committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-4/main into branch-itinerary-tests

# Conflicts:
#	src/main/java/seedu/address/logic/commands/itinerary/events/edit/EditEventFieldCommand.java
#	src/main/java/seedu/address/model/itinerary/event/Event.java
#	src/main/java/seedu/address/model/itinerary/event/EventList.java
#	src/main/java/seedu/address/storage/JsonAdaptedEvent.java
#	src/test/java/seedu/address/testutil/TripBuilder.java
#	src/test/java/seedu/address/testutil/TypicalTrips.java

teoha committed:
Added:
1. Itinerary storage tests
Changed:
1. Storage bugs

ang-zeyu committed:
Merge pull request #163 from teoha/branch-itinerary-tests

Important: Major bug fixes and itinerary tests

ang-zeyu committed:
Changes:
- minor ui tweaks

Fixes:
- fixes #128 - needing to click done button twice in diary
- fixes #118 - long dates overflowing trip card container
- Workaround for listcell getIndex() having phantom out of bounds cell indexes after entry switch

ang-zeyu committed:
Merge branch 'master' into branch-diary

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

ang-zeyu committed:
Small diary ui tweaks

ang-zeyu committed:
Changes:
- Some minor performance optimizations and code reorganization in DiaryPage
- Minor uml diagram documentation tweak
- Changed CreateDiaryEntryCommand not to show edit box by default

ang-zeyu committed:
Merge branch 'branch-diary' into branch-project-portfolio

arjunwangty committed:
Update tests

arjunwangty committed:
Merge branch 'Branch-Expense-Manager' into Branch-Developer-Guide

teoha committed:
Changed:
1. Made tumbnail text more visible
2. Fixed event and day list bugs

ang-zeyu committed:
Changes:
- Documentation update for project portfolio page, user guide and developer guide
- Update some UML diagrams for better formatting

ang-zeyu committed:
Changes:
- Update user guide images with line numbers

ang-zeyu committed:
Merge branch 'branch-project-portfolio' into branch-diary

teoha committed:
Added:
1. Derrick's PPP

Commit:
1. Prevented Duplicate Inventory Items from Being Created (Inventory View Page will only view unique inventories, and each Event will have unique inventories)
2. Added Inventory List View to the EventsPage
3. Fixed Some Bugs with JSonAdaptedInventory & the Event class
4. Modified the Done Command to Allow Both Checking and Unchecking
5. Improved the UI of Inventory View Page (by adding numbering and by displaying checked (done) inventory items as being ticked in a checkbox)

SakshiPradyumn committed:
Merge branch 'master' into branchBooking

Resolved all

SakshiPradyumn committed:
Merge branch 'master' into branchBooking

Resolved yet again

Commit:
Added JUnit Testing for the Inventory Feature

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-4/main into BranchForPretripItinerary (All Conflicts Removed)

# Conflicts:
#	src/main/java/seedu/address/logic/commands/itinerary/events/edit/DoneEditEventCommand.java
#	src/main/java/seedu/address/logic/commands/itinerary/events/edit/EditEventFieldCommand.java
#	src/main/java/seedu/address/logic/parser/itinerary/ItineraryParserUtil.java
#	src/main/java/seedu/address/logic/parser/itinerary/eventview/edit/EditEventFieldParser.java
#	src/main/java/seedu/address/model/itinerary/event/Event.java
#	src/main/java/seedu/address/model/itinerary/event/EventList.java
#	src/main/java/seedu/address/storage/JsonAdaptedEvent.java
#	src/main/java/seedu/address/ui/itinerary/EditEventPage.java
#	src/main/java/seedu/address/ui/itinerary/EventsPage.java
#	src/main/resources/view/itinerary/events/EventsPage.fxml
#	src/test/java/seedu/address/testutil/EventBuilder.java

Commit:
Added new JUnit Test Cases for the Inventory Feature

teoha committed:
Changed:
1. Itinerary introduction

teoha committed:
Added:
1. All missing javadocs for itinerary feature

teoha committed:
Changed:
1. Checkstyle errors

teoha committed:
Changed:
1. File name for PPP

teoha committed:
Merge branch 'branch-itinerary-ui-1.4' into branch-itinerary-guides

Commit:
Corrected all Check-styles for Regular & Test Classes

Commit:
Allowed Inventory Items to be Deleted using the GUI in the EditEventPage

Commit:
Improved the Success Message for adding or deleting an inventory in the EditEventsPage

ang-zeyu committed:
Merge pull request #166 from eizon05/BranchForPretripItinerary

Added the following improvements:

ang-zeyu committed:
Merge branch 'master' into branch-itinerary-ui-1.4

ang-zeyu committed:
Merge pull request #165 from teoha/branch-itinerary-ui-1.4

Fix for event and day list changing index

Commit:
Styled the Inventory List View (using css) & Resized the Inventory Page's Table View

SakshiPradyumn committed:
Not working Bookings

ang-zeyu committed:
Changes:
- project portfolio tweaks
- fix not being able to scroll all the way to the bottom in EditTripPage

ang-zeyu committed:
Changes:
- more documentation tweaks

ang-zeyu committed:
Merge branch 'master' into branch-diary
- Update EditTripPage fix for not being able to scroll to bottom of the form

# Conflicts:
#	src/main/java/seedu/address/logic/parser/itinerary/ItineraryParserUtil.java
#	src/main/resources/view/trips/EditTripPage.fxml

ang-zeyu committed:
Changes:
- More documentation tweaks
- Tweak DoneEditEntryTextCommand error message

teoha committed:
Merge pull request #164 from ang-zeyu/branch-diary

Branch diary

ang-zeyu committed:
Code Cleanup:
- General code cleanup (access modifiers, unneeded declarations, etc.)
- Remove unneeded isGuiLocked from UserPrefs
- Fix DiaryGalleryCard equals incorrectly comparing type equality to PersonCard
- update MainApp version number
- some more documentation tweaks

ang-zeyu committed:
Update MainApp.java version number

ang-zeyu committed:
Merge branch 'master' into branch-diary

teoha committed:
Added:
1. PPP
2. UG
3. DG

arjunwangty committed:
Update DG

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

# Conflicts:
#	src/test/java/seedu/address/model/ModelTestUtil.java

SakshiPradyumn committed:
Incomplete PR for Derrick to check ArgumentTokenizer

SakshiPradyumn committed:
Merge branch 'master' into branchBooking

Resolved conflicts

ang-zeyu committed:
Changes:
- More documentation tweaks

ang-zeyu committed:
Changes:
- Final tweaks to diary documentation

arjunwangty committed:
Fix checkstyle, update UG

SakshiPradyumn committed:
Committing with some resolved

teoha committed:
Changed:
1. Checkstyle errors

teoha committed:
Changed:
Checkstyle errors

SakshiPradyumn committed:
Checkstyle Main corrected

teoha committed:
Merge branch 'master' into branch-itinerary-guides

SakshiPradyumn committed:
Resolved TripBuilder

ang-zeyu committed:
Quick update to common developer guide sections

teoha committed:
Merged conflicts

teoha committed:
Merge pull request #161 from SakshiPradyumn/branchBooking

Version 1 Booking

teoha committed:
Merge branch 'master' into branch-itinerary-guides

ang-zeyu committed:
Quick change to INVALID_COMMAND_TYPE error message for #126

ang-zeyu committed:
Merge branch 'master' into branch-diary

arjunwangty committed:
Update PPP

teoha committed:
Merge pull request #170 from teoha/branch-itinerary-guides

Branch itinerary guides

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

# Conflicts:
#	docs/UserGuide.adoc

teoha committed:
Merge branch 'master' into branch-diary

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

# Conflicts:
#	docs/DeveloperGuide.adoc

aang114 committed:
Merge branch 'master' into BRANCHTOPRFROM

Commit:
Added newline to CSS

Commit:
Merge branch 'BRANCHTOPRFROM' of https://github.com/eizon05/main into BRANCHTOPRFROM

arjunwangty committed:
Update PPP, Fix DG

arjunwangty committed:
Merge pull request #169 from arjunwangty/Branch-Developer-Guide

Update Developer Guide/ User Guide

Commit:
Updated DG on the Inventory Feature

Commit:
Added more Diagrams for DG

arjunwangty committed:
Fix PPP

Commit:
Updated UG to elaborate more on the Inventory Feature

Commit:
Add eventsPage.png

arjunwangty committed:
Merge branch 'master' into Branch-Developer-Guide

Commit:
Correcting grammer mistakes for UG

arjunwangty committed:
Merge pull request #172 from arjunwangty/Branch-Developer-Guide

Fix PPP / DG

ang-zeyu committed:
Merge branch 'master' into BRANCHTOPRFROM

teoha committed:
Changes:
1. page ui changes

ang-zeyu committed:
Merge pull request #171 from eizon05/BRANCHTOPRFROM

Styled the Inventory List View (using css) & Resized the Inventory Pa…

ang-zeyu committed:
Merge branch 'master' into branch-itinerary-guides

ang-zeyu committed:
Merge pull request #173 from teoha/branch-itinerary-guides

Ui last changes

teoha committed:
Changed:
1. UG help
2. DG appendix cleanup

ang-zeyu committed:
Massive old code cleanup

ang-zeyu committed:
Merge branch 'master' into branch-diary

ang-zeyu committed:
Merge branch 'branch-diary' of https://github.com/ang-zeyu/main into branch-diary

teoha committed:
Merge branch 'branch-itinerary-guides' of https://github.com/teoha/main into branch-itinerary-guides

ang-zeyu committed:
Fix test checkstyle

teoha committed:
Added:
Short ug for exit

ang-zeyu committed:
Merge pull request #168 from ang-zeyu/branch-diary

Code Cleanup:

teoha committed:
Merge branch 'master' into branch-itinerary-guides

ang-zeyu committed:
Merge pull request #174 from teoha/branch-itinerary-guides

Branch itinerary guides

Commit:
Updating UG (I had put the description of the Event Inventory List in the Trips Section by mistake)

aang114 committed:
Update UserGuide.adoc

ang-zeyu committed:
Merge pull request #175 from eizon05/BranchToUpdateUG

Updating UG (I had put the description of the Event Inventory List in…

SakshiPradyumn committed:
Booking Feature User Guide added

SakshiPradyumn committed:
Developer Guide Booking Feature.

SakshiPradyumn committed:
Merge pull request #176 from SakshiPradyumn/branchDocuments

Booking Feature User Guide added

SakshiPradyumn committed:
Images added

SakshiPradyumn committed:
Merge branch 'master' into branchDocuments

SakshiPradyumn committed:
Merge pull request #177 from SakshiPradyumn/branchDocuments

Images added

SakshiPradyumn committed:
Images UG added.

SakshiPradyumn committed:
Removing broken images

SakshiPradyumn committed:
Merge branch 'master' into branchDocuments

SakshiPradyumn committed:
Merge pull request #178 from SakshiPradyumn/branchDocuments

Removing broken images

SakshiPradyumn committed:
Commit, Push

Commit:
Uploading my PPP (I had already uploaded it on Monday night on LumiNUS, but I had forgot to upload it on the GitHub Team Repo)

Commit:
Fixing check-style errors

arjunwangty committed:
Merge pull request #180 from eizon05/BranchToUploadPPP

Uploading my PPP (I had already uploaded it on Monday night on LumiNU…

SakshiPradyumn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T11-4/main

SakshiPradyumn committed:
Commit PPP
(already done on Tuesday, submitted on LumiNUS as well)

SakshiPradyumn committed:
EOF added to PPP

SakshiPradyumn committed:
Merge pull request #181 from SakshiPradyumn/branchPPPUpload

Branch PPP upload
